formula-android/src/main/java/com/instacart/formula/android/internal/FeatureBinding.kt

```diff
@@ -33,6 +32,7 @@ internal class FeatureBinding<in Component, in Dependencies, in Key : FragmentKe
                         if (binds(key)) {
                             Action.onData(fragmentId).onEvent {
                                 transition {
+                                    // TODO: should this happen on the main thread? It needs to be available to main thread
```

Comment:
This is a non-blocking todo which I will address in a follow-up

Reply:
Moved main thread scheduler to `FragmentFlowStore` so this will always be executed on the main thread.